setup.py

```diff
@@ -9,6 +9,6 @@
     url='https://github.com/bastibe/jbof',
     license='GPL v3',
     py_modules=['runforrest'],
-    install_requires=['numpy'],
+    install_requires=['numpy', 'json', 'soundfile', 'scipy'],
```

Comment:
No need to add `json`, as it is part of the standard lib.

Reply:
DONE in 4178140